tests/std/include/range_algorithm_support.hpp

```diff
@@ -473,96 +492,141 @@ namespace test {
         }
 
         // input iterator operations:
-        constexpr void operator++(int) & noexcept requires std::is_same_v<Category, input> {
+        constexpr void operator++(int) & noexcept
+            requires std::is_same_v<Category, input>
+        {
             ++ptr_;
         }
 
-        [[nodiscard]] friend constexpr Element&& iter_move(iterator const& i) requires at_least<input> {
+        [[nodiscard]] friend constexpr Element&& iter_move(iterator const& i)
+            requires at_least<input>
+        {
             return std::move(*i.ptr_);
         }
 
         friend constexpr void iter_swap(iterator const& x, iterator const& y) noexcept(
-            std::is_nothrow_swappable_v<Element>) requires at_least<input> && std::swappable<Element> {
+            std::is_nothrow_swappable_v<Element>)
+            requires at_least<input> && std::swappable<Element>
+        {
             ranges::swap(*x.ptr_, *y.ptr_);
         }
 
         // forward iterator operations:
-        constexpr iterator operator++(int) & noexcept requires at_least<fwd> {
+        constexpr iterator operator++(int) & noexcept
+            requires at_least<fwd>
+        {
             auto tmp = *this;
             ++ptr_;
             return tmp;
         }
 
         // sentinel operations (implied by forward iterator):
-        iterator(iterator const&) requires(to_bool(Eq))            = default;
-        iterator& operator=(iterator const&) requires(to_bool(Eq)) = default;
-
-        constexpr operator Consterator() const& noexcept requires(to_bool(Eq)) {
+        iterator(iterator const&)
+            requires (to_bool(Eq))
+        = default;
+        iterator& operator=(iterator const&)
+            requires (to_bool(Eq))
+        = default;

```

Comment:
I could hold my noise for the single defaulted function on 401-403, but this is too much. I'll push `// clang-format off`s for both here and there.